packages/eslint-config-forumone-es5/README.md

```
@@ -6,9 +6,27 @@ Our JavaScript code style conventions are based on:
 - [TypeScript ESLint](https://typescript-eslint.io/)
 - [Prettier](https://prettier.io/docs/related-projects#eslint-integrations)
 - [Prettier Plugin: Organize Imports](https://github.com/simonhaenisch/prettier-plugin-organize-imports)
+- [Storybook ESLint](https://storybook.js.org/docs/configure/integration/eslint-plugin)
 with customizations for specific environments and team practices.
 
 Usage
 -----
 1. Run `npm install eslint-config-forumone-es5`.
 2. Add `extends: 'forumone-es5'` to your ESLint configuration file.
+
+### Storybook
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)